File: CS/DXWebApplication/DXWebApplication/Controllers/HomeController.cs
using DevExpress.Web;
using DevExpress.Web.Mvc;
using DXWebApplication;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using System.Web.UI.WebControls;

namespace DXWebApplication {
    [ValidateInput(false)]
    public class HomeController : Controller {
        public string Name { get { return "Common"; } }

        public ActionResult Index() {
            return RedirectToAction("ModelValidation");
        }
        public ActionResult jQueryValidation() {
            return View("jQueryValidation", new JQueryValidationData());
        }
        [HttpPost]
        public ActionResult jQueryValidation(JQueryValidationData validationData) {
            if(Request.Params["btnUpdate"] == null) {
                ModelState.Clear();
                return View("jQueryValidation", validationData);
            }

            if(ModelState.IsValid) {
                return View("_ValidationSuccess");
            }
            return View("jQueryValidation", validationData);
        }
        public JsonResult CheckReleaseDate(DateTime? ReleaseDate) {
            return Json(ReleaseDate != null && ReleaseDate >= DateTime.Today, JsonRequestBehavior.AllowGet);
        }
    }

    
}

File: VB/DXWebApplication/DXWebApplication/Controllers/HomeController.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc

Namespace DXWebApplication
    <ValidateInput(False)> _
    Public Class HomeController
        Inherits Controller

        Public ReadOnly Property Name() As String
            Get
                Return "Common"
            End Get
        End Property

        Public Function Index() As ActionResult
            Return RedirectToAction("ModelValidation")
        End Function
        Public Function jQueryValidation() As ActionResult
            Return View("jQueryValidation", New JQueryValidationData())
        End Function
        <HttpPost> _
        Public Function jQueryValidation(ByVal validationData As JQueryValidationData) As ActionResult
            If Request.Params("btnUpdate") Is Nothing Then
                ModelState.Clear()
                Return View("jQueryValidation", validationData)
            End If

            If ModelState.IsValid Then
                Return View("_ValidationSuccess")
            End If
            Return View("jQueryValidation", validationData)
        End Function
        Public Function CheckReleaseDate(ByVal ReleaseDate As Date) As JsonResult
            Return Json(ReleaseDate >= Date.Today, JsonRequestBehavior.AllowGet)
        End Function
    End Class


End Namespace